services/ows_refactored/dev_af_ows_root_cfg.py

```diff
@@ -73,7 +73,7 @@
     },  # END OF global SECTION
     "wms": {
         # Config for WMS service, for all products/layers
-        "s3_url": "https://data.digitalearth.africa",
+        # "s3_url": "https://data.digitalearth.africa",
```

Comment:
can i remove this?

Reply:
The s3_url and s3_bucket entries are only used for converting the links to the underlying data files into pretty http links for the GetFeatureInfo responses.

Removing will leave you with broken URLs - but that looks like something I'm going to have handle within OWS.